.github/workflows/release.yml

```
@@ -43,12 +46,20 @@ jobs:
         with:
           token: ${{ secrets.GH_REPO_TOKEN }}
 
-  deploy-svn:
-    name: 'Deploy with svn'
-    needs: release
-    if: needs.release.outputs.released == 'true'
-    uses: ./.github/workflows/svn-deploy.yml
-    secrets: inherit
+      - name: 'Create dist folder'
+        if: steps.release.outputs.released == true
```

```suggestion
        if: steps.release.outputs.released == 'true'
```

https://docs.github.com/en/actions/learn-github-actions/expressions#functions